app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -66,6 +66,8 @@ def start
     end
 
     queue_signal(:pod_wait)
+  rescue
+    queue_abort("unexpected exception, see log")
```

Comment:
I see in #8743 you're using this method as a general way to indicate a failed job (i.e., "no image found").  So, just allow the exception to be nil in `queue_abort` and if it's provided include a traceback or something like that in the logs.

Just a thought.